CMakeLists.txt

```diff
@@ -51,5 +54,26 @@ target_compile_options(MolSim
 )
 
 list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake/modules)
+#Option to build Documentation. Can be disabled by calling cmake -D BUILD_DOC=OFF ..
+option(BUILD_DOC "Build documentation" ON)
+if(BUILD_DOC)
+include(doxygen)
```

Comment:
The idea of `include` statements is that not everything is placed in one file.

Everything below this include should go to `<repo-root>/cmake/modules/doxygen.cmake`

Reply:
If you move this in the file tree, you can change `${CMAKE_CURRENT_SOURCE_DIR}/Doxyfile` to something like `${PROJECT_SOURCE_DIR}/Doxyfile`.

Always have a look at the [docs](https://cmake.org/cmake/help/v3.0/variable/PROJECT_SOURCE_DIR.html#variable:PROJECT_SOURCE_DIR)